package.json

```diff
@@ -8,9 +8,12 @@
     "build": "parcel build index.html --no-cache",
     "lint": "eslint 'src/**/*.js' --quiet --fix",
     "clean": "rimraf ./dist ./.cache",
-    "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js"
+    "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js",
+    "test": "jest",
+    "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@testing-library/dom": "^7.5.7",
```

Comment:
This should be added to dev dependencies, as testing-library is being included from a cdn at the moment.

See comment here for why it is that way:

https://github.com/smeijer/testing-playground/blob/6420421a24e8792f30fccc8482d04d732125d30a/src/parser.js#L5:L11

Reply:
I have added here for a future implementation. I move it in dev dep

Reply:
Yeah, I expected that. But I was afraid that it will end up in the production bundle. Thanks.